src/lib/theme/colors.ts

```diff
@@ -139,29 +139,32 @@ const palette: Readonly<Record<string, Record<Shade, HexColor>>> = {
     950: '#461B02',
     DEFAULT: '#F8A208',
   },
-};
+} as const satisfies Readonly<Record<string, PaletteColor>>;
 
 type Palette = typeof palette;
 
-export const getColor = <C extends keyof Palette>(
-  color: C,
-  shade: keyof Palette[C] = 'DEFAULT',
+export const getColor = (
+  color: keyof Palette,
+  shade: Shade = 'DEFAULT',
```

Comment:
I don't think the typing was working properly before, now `getColor` is strongly typed
<img width="1840" alt="Screenshot 2024-02-23 at 11 09 20 AM" src="https://github.com/temporalio/ui/assets/11775628/ef85e7da-5a46-4ca2-a8ab-ddbc704de8bc">